metaflow/plugins/argo/argo_live_run.py

```diff
@@ -166,9 +164,9 @@ def _wait(self, wait_timeout) -> None:
                     f" minutes out of a possible {wait_timeout}"
                 )
             loop_counter += 1
```

Comment:
Can you also add doc string for why `loop_counter` is needed? I think your design is fine for checking status more frequently than status report, but you want to make your design explicit to developers who do not share the same context.

Reply:
I added comment:

> tracks loop count so status printed every minute

Is this ok?

Reply:
I'd try to make the design / intention more explicit by either adding more doc str or renaming a few variable. 

Maybe instead of say sleep(5), you can define local var called `status_check_interval: int = 5`, and a local var `status_report_interval = 60` to be compared with `time.time() - start_time`.